src/VDB/Spider/Downloader/Downloader.php

```diff
@@ -86,12 +86,12 @@ public function isDownLoadLimitExceeded()
     /**
      * A shortcut for EventDispatcher::dispatch()
      *
+     * @param Event $event
      * @param string $eventName
-     * @param null|Event $event
      */
-    private function dispatch($eventName, Event $event = null)
+    private function dispatch(Event $event, $eventName)
```

Comment:
In both cases (also in Spider) this should be `private function dispatch(GenericEvent $event, $eventName)`.

Reply:
Okay, I've updated both cases. Could you verify it?